ci/long_running_tests/workloads/many_drivers.py

```diff
@@ -57,8 +57,8 @@ def method(self):
 for _ in range(5):
     for i in range(num_nodes):
         assert (ray.get(
-            f._remote(args=[], kwargs={{}}, resources={{str(i): 1}})) == 1)
-        actor = Actor._remote(args=[], kwargs={{}}, resources={{str(i): 1}})
+            f.options(resources={{str(i): 1}})) == 1).remote(args=[], kwargs={{}})
```

Comment:
Don't need to use `args=[], kwargs={}` anymore if using regular remote call. Can just pass them like a normal function call. Could you update all of the callsites?

Reply:
Ok, I will resolve all the problems soon.